### ISSUE TITLE
Improve validation of non-mandatory columns

### ISSUE THREAD

**Issue by tudorgroza** [EBISPOT/gwasdepo-template-validator#2]
As per: https://github.com/EBISPOT/goci/issues/128, enforce field validation even if the column is not mandatory, but values are present

**Comment by tudorgroza** [comment]
@ljwh2 @eks-ebi - this is deployed in sandbox and ready for testing.

**Comment by twhetzel** [comment]
While testing another update, I also wound up testing this change and can confirm when a value is present for "Array Manufacturer" that is not one of the accepted values the validation returns this notification:
```
Studies: Column 'Array manufacturer' in row(s) [6] contains incorrect value. Accepted values are: 'Illumina; Perlegen; Affymetrix'
```

**Comment by tudorgroza** [comment]
Thank you @twhetzel

**Comment by eks-ebi** [comment]
Tested today 2020-07-27.
Test submission: http://193.62.54.159/gwas/deposition/submission/5f1edecb86621e00017f0213

I introduced errors in these mandatory fields:

1. Genotyping technology -> I got a sensible error detailing the accepted values
2. Stage -> I got a sensible error detailing the accepted values
3. Ancestry category -> I got a sensible error detailing the accepted values
4. Country of recruitment -> I got a sensible error detailing the accepted values

And these optional fields:

1. Array manufacturer -> I got a sensible error detailing the accepted values
2. Summary statistics assembly -> I got a sensible error detailing the accepted values

So all OK!
